TCAT/Controllers/InformationViewController.swift

```diff
@@ -42,138 +43,137 @@ class InformationViewController: UIViewController, UITableViewDataSource, UITabl
 
     ]
 
+    private let descriptionLabelSize = CGSize(width: 167, height: 34)
+    private let descriptionLabelTopOffset: CGFloat = 12
+    private let headerWidth = UIScreen.main.bounds.width
+    /** Represents the total height of the headerView given the width of the phone screen. Most of the variables
+     used are static except for the tcat image height, which is dependent on the width of the user's screen.
+     The extra 37 is used for bottom inset padding for the headerView, but I felt it would be unnecessary to
+     create a variable for it */
+    private var headerHeight: CGFloat {
+        return tcatImageTopOffset + tcatImageSize.height + titleLabelTopOffset + titleLabelSize.height + descriptionLabelTopOffset + descriptionLabelSize.height + 37
+    }
+    private var tcatImageSize: CGSize { return CGSize(width: headerWidth-80, height: (headerWidth - 80) * 0.4) }
+    private let tcatImageTopOffset: CGFloat = 44
+    private let titleLabelSize = CGSize(width: 240, height: 20)
+    private let titleLabelTopOffset: CGFloat = 44
+
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        let payload = AboutPageOpenedPayload()
+        Analytics.shared.log(payload)
+
         title = Constants.Titles.aboutUs
 
         view.backgroundColor = Colors.backgroundWash
         navigationController?.navigationBar.tintColor = Colors.primaryText
 
-        view.addSubview(hiddenLabel)
-        view.addSubview(tcatImage)
-        view.addSubview(titleLabel)
-        view.addSubview(descriptionLabel)
+        setupTableView()
+
+        setupConstraints()
+    }
+
+    func setupTableView() {
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: Constants.Cells.informationCellIdentifier)
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.backgroundColor = Colors.backgroundWash
+        tableView.separatorColor = Colors.dividerTextField
+        tableView.showsVerticalScrollIndicator = false
+
+        setupHeaderView()
+        tableView.tableHeaderView = headerView
+
         view.addSubview(tableView)
+    }
+
+    func setupHeaderView() {
+        headerView = UIView(frame: CGRect(x: 0, y: 0, width: headerWidth, height: headerHeight))
```

Comment:
i think it might be better to just create a custom UIView `InformationHeaderView` and then you just use that inside here instead of having all the views in here as well. want to avoid big view controllers as much as possible